src/main.cc

```diff
@@ -326,7 +326,14 @@ void Application::after_run() {
 
 Application::~Application() {
     if (use_profiler && Profiler::is_initialized()) {
-        Profiler::instance()->output(PETSC_COMM_WORLD);
+        // log profiler data to this stream
+        Profiler::instance()->output (PETSC_COMM_WORLD);
+
+        // call python script which transforms json file at given location
+        Profiler::instance()->transform_profiler_data (".csv", "CSVFormatter");
```

Comment:
Comment out CSV output for now, later we may allow selection of profiler output instead of no_profile comad line parameter.